[devtools-translate] Chinese - What's New in DevTools (Chrome 107) (#3997)
GoogleChrome/developer.chrome.com: site/zh/blog/new-in-devtools-107/index.md

  - new-in-devtools
  - devtools
  - chrome-107

> draft: flase

Reviewer: ```suggestion
```
Remove it will do

  - new-in-devtools
  - devtools
  - chrome-107
---

*感谢 [JimLim]([https://www.linkedin.com/in/zui-yong-poong-1b507b14/](https://www.linkedin.com/in/jim-lim-539a5638)) 提供的翻译*

{% include 'partials/devtools/zh/banner.md' %}

> 

Reviewer: ```suggestion
{% YouTube id='1uwv6HbR8HU' %}
```

<!-- Content starts here -->

## 客制化 DevTools 的键盘快捷键 {: #shortcuts }

您现在可以在 DevTools 里客制化您喜爱的键盘快捷键。

> <!-- 去到**Settings(设置)** > **Shortcuts(快捷键）**, 将鼠标徘徊在命令上并点击**Edit（编辑）** 按钮 (钢笔图示）来客制化键盘快捷键。您也可以创建组合键 You can create chords (a.k.a 多键点击快捷键)。  -->

Reviewer: ```suggestion
去到**Settings(设置)** > **Shortcuts(快捷键）**, 将鼠标徘徊在命令上并点击**Edit（编辑）** 按钮 (钢笔图示）来客制化键盘快捷键。您也可以创建组合键 You can create chords (a.k.a 多键点击快捷键)。
```

Reviewer: Please remove all the commented bracket when translate

> 

Reviewer: Please update `whats-new.md` file as well

<!-- Content starts here -->

> <!-- ## 客制化DevTools的键盘快捷键 {: #shortcuts } -->

Reviewer: ```suggestion
## 客制化 DevTools 的键盘快捷键 {: #shortcuts }
```
- Always add a space between English word and Chinese.
- Remove the `<!-- -->` because this is the syntax for making it a comment.

Please help to apply both the above for the whole article.

> <!-- 您现在可以在DevTools中客制化你喜爱的命令的键盘快捷键 -->

Reviewer: ```suggestion
您现在可以在 DevTools 里客制化您喜爱的键盘快捷键。
```

Always consistently use “您” for the article

<!-- Content starts here -->

## 客制化 DevTools 的键盘快捷键 {: #shortcuts }

您现在可以在 DevTools 里客制化您喜爱的键盘快捷键。

去到 **Settings（设置）** > **Shortcuts（快捷键）**, 将鼠标徘徊在命令上并点击 **Edit（编辑）** 按钮 (钢笔图示）来客制化键盘快捷键。您也可以创建组合键（a.k.a. 多键点击快捷键）。

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/973EfWpxwGOdEF1nN1vv.png", alt="客制化 DevTools 的键盘快捷键。", width="800", height="516" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/d061128ff63a97ab2c6c0d2b5e655e6fcbed829c #}

Chromium issues: [1335274](https://crbug.com/1335274), [174309](https://crbug.com/174309)

> ## 使用键盘快捷键对光亮与暗黑主题进行快速切换。 {: #toggle-themes }

Reviewer: ```suggestion
## 使用键盘快捷键对浅色与深色主题进行快速切换。 {: #toggle-themes }
```

> 设置一个键盘快捷键来对光亮与暗黑主图便利地进行快速切换。这个行动的快捷键并没有被系统预设。 

Reviewer: ```suggestion
设置一个键盘快捷键来对浅色与深色主图便利地进行快速切换。这个行动的快捷键并没有被系统预设。 
```

<!-- Content starts here -->

## 客制化 DevTools 的键盘快捷键 {: #shortcuts }

您现在可以在 DevTools 里客制化您喜爱的键盘快捷键。

> 在 **Settings（设置）** > **Shortcuts（快捷键）**, 将鼠标徘徊在命令上并点击 **Edit（编辑）** 按钮 (钢笔图示）来客制化键盘快捷键。您也可以创建组合键（a.k.a. 多键点击快捷键）。

Reviewer: ```suggestion
在 **Settings（设置）** > **Shortcuts（快捷键）**, 将鼠标悬停在命令上并点击 **编辑** 按钮 (钢笔图示）来客制化键盘快捷键。您也可以创建组合键（a.k.a. 多键点击快捷键）。
```

> 去到 **Settings（设置）** > **Shortcuts（快捷键）**, 将鼠标徘徊在命令上并点击 **Edit（编辑）** 按钮 (钢笔图示）来客制化键盘快捷键。您也可以创建组合键（a.k.a. 多键点击快捷键）。

Reviewer: ```suggestion
在 **Settings（设置）** > **Shortcuts（快捷键）**, 将鼠标徘徊在命令上并点击 **Edit（编辑）** 按钮 (钢笔图示）来客制化键盘快捷键。您也可以创建组合键（a.k.a. 多键点击快捷键）。
```

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/973EfWpxwGOdEF1nN1vv.png", alt="客制化 DevTools 的键盘快捷键。", width="800", height="516" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/d061128ff63a97ab2c6c0d2b5e655e6fcbed829c #}

Chromium 议题: [1335274](https://crbug.com/1335274), [174309](https://crbug.com/174309)

> ## 使用键盘快捷键对浅色与深色主题进行快速切换。 {: #toggle-themes }

Reviewer: ```suggestion
## 使用键盘快捷键对浅色与深色主题进行快速切换 {: #toggle-themes }
```

> 设置一个键盘快捷键来对浅色与深色主图便利地进行快速切换。这个行动的快捷键并没有被系统预设。 

Reviewer: ```suggestion
```

This line is not suppose to be here, always translate under the original English sentence

<!-- Configure a keyboard shortcut to toggle [light and dark themes](/docs/devtools/rendering/emulate-css/#emulate-css-media-feature-prefers-color-scheme) conveniently. By default, the action doesn’t map to any keyboard shortcut. -->

> 

Reviewer: ```suggestion

设置一个快捷键来对浅色与深色便利地进行快速切换。这个快捷键并没有系统默认。 
```
What is 主图 means？

> {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/7oGdE2eRsgwokWXW9XvA.png", alt="使用键盘快捷键对光亮与暗黑主题进行快速切换。", width="800", height="576" %}

Reviewer: ```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/7oGdE2eRsgwokWXW9XvA.png", alt="使用键盘快捷键对浅色与深色主题进行快速切换。", width="800", height="576" %}
```

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/4853b34457f43e41ae9cebc7dfc97c0b734f463a #}
{# https://chromium.googlesource.com/devtools/devtools-frontend/+/029ac9db0b7e7d08945bcf7a16b407bde50183a1 #}

Chromium 议题: [1280398](https://crbug.com/1280398), [1226363](https://crbug.com/1226363)

> ## 高亮内存检查器里的 C/C++ 对象。 {: #memory } 

Reviewer: ```suggestion
## 高亮内存检查器里的 C/C++ 对象 {: #memory } 
```

> ## 强标记忆检查器里的C/C++对象。 {: #memory } 

Reviewer: ```suggestion
## 高亮内存检查器里的 C/C++ 对象。 {: #memory } 
```
Please check the correct term in DevTools.

<!-- The [Memory Inspector](/docs/devtools/memory-inspector/) highlights all the bytes of a C/C++ memory object. -->

> 辨识围绕在一个对象字节中的网络组件记忆体是一个用户痛点，您必须知道对象的体积以及从对象初始的时候开始计算字节数。 

Reviewer: ```suggestion
之前，开发者难以在 WebAssembly 内存中辨识一个对象的 bytes 数组。开发者必须知道对象的体积以及手动计算对象的 bytes 数组。 
```

> 有了这个新功能,  它会帮助您指示他们从围绕的记忆体中分离， 参看 [延伸为了C/C++ 纠错而发生的机器检查器](/blog/memory-inspector-extended-cpp/) 来学习更多的改善。 

Reviewer: ```suggestion
有了这个新功能,  它会帮助您指示他们从围绕的记忆体中分离， 参看[延伸为了C/C++ 纠错而发生的机器检查器](/blog/memory-inspector-extended-cpp/) 来学习更多的改善。 
```

> {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/zqOv2zJTc8ucoeDmQiTo.png", alt="强标记忆检查器里的C/C++对象。", width="800", height="527" %}

Reviewer: ```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/zqOv2zJTc8ucoeDmQiTo.png", alt="高亮记忆检查器里的C/C++对象。", width="800", height="527" %}
```

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/d5f3befb47eaaa373d697b42dec6f179baf9d42c #}
{# https://chromium.googlesource.com/devtools/devtools-frontend/+/c4e6bdb4321cbc0b783647e855a616096beaabfd #}

Chromium 议题: [1336568](https://crbug.com/1336568)

> ## 支持 HAR 导入的完整初始者的资讯。{: #har } 

Reviewer: ```suggestion
## 支持 HAR 导入的完整启动器的资讯。{: #har } 
```

> ## 支持HAR 导入的完整初始者的资讯。{: #har } 

Reviewer: ```suggestion
## 支持 HAR 导入的完整初始者的资讯。{: #har } 
```

> [HAR import] 的完整 **Initiator（初始者)** 讯息现在可以可见(/docs/devtools/network/reference/#save-as-har). 在之前的版本中, **Network（网络）** 界面只显示导入时的部分初始者讯息。 

Reviewer: ```suggestion
Full **Initiator** information is available now for [HAR import](/docs/devtools/network/reference/#save-as-har). Previously, the **Network** panel only shows partial initiator information during import.

现在，当您[导入 HAR 文件](/docs/devtools/network/reference/#save-as-har)时，DevTools 会呈现完整的启动器讯息。之前，**网络** 面板只显示部分的讯息。 
```

> [HAR import]的完整**Initiator（初始者)** 讯息现在可以可见(/docs/devtools/network/reference/#save-as-har). 在之前的版本中, **Network（网络）** 界面只显示导入时的部分初始者讯息。 

Reviewer: ```suggestion
[HAR import] 的完整 **Initiator（初始者)** 讯息现在可以可见(/docs/devtools/network/reference/#save-as-har). 在之前的版本中, **Network（网络）** 界面只显示导入时的部分初始者讯息。 
```

> 完整的初始者讯息将帮助开发者检索网络请求的源头并辨别和网络相关的问题。  

Reviewer: ```suggestion
完整的启动器讯息将帮助开发者检索网络请求的源头并调试网络相关的问题。  
```

> {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/cthh3ZrpDwo4LJiaY4Uo.png", alt="支持 HAR 导入的完整初始者的资讯。", width="800", height="376" %}

Reviewer: ```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/cthh3ZrpDwo4LJiaY4Uo.png", alt="导入 HAR 文件时完整显示启动器资讯。", width="800", height="376" %}
```

> {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/cthh3ZrpDwo4LJiaY4Uo.png", alt="支持HAR 导入的完整初始者的资讯。", width="800", height="376" %}

Reviewer: ```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/cthh3ZrpDwo4LJiaY4Uo.png", alt="支持 HAR 导入的完整初始者的资讯。", width="800", height="376" %}
```

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/3a659b0711f52a2e200395b85f16ed9f266d1571 #}

Chromium 议题: [1343185](https://crbug.com/1343185)

> ## 点击 `Enter（回车键）`后开始 DOM 搜索。 {: #search-type } 

Reviewer: ```suggestion
## 点击 `Enter（回车键）`后开始 DOM 搜索 {: #search-type } 
```

> ## 点击`Enter（回车键）`后开始DOM搜索。 {: #search-type } 

Reviewer: ```suggestion
## 点击 `Enter（回车键）`后开始 DOM 搜索。 {: #search-type } 
```

> 您现在可以禁止**Search as you type（输入即搜索** 设置来启动点击<kbd>Enter</kbd>后开始DOM搜索。  

Reviewer: ```suggestion
您现在可以禁止 **Search as you type（输入即搜索** 设置来启动点击 <kbd>Enter</kbd> 后开始DOM搜索。  
```

> 在**Elements（元素）**界面, 使用<kbd>Control</kbd> 或 <kbd>Command</kbd> + <kbd>F</kbd>来对搜索栏进行快速切换。 当您在搜索栏中输入一个查询时, DOM数会跳跃至第一个符合条件的元素并将至设置为预设。 

Reviewer: ```suggestion
在**元素**面板, 使用 <kbd>Control</kbd> 或 <kbd>Command</kbd> + <kbd>F</kbd> 来打开搜索栏。 每当您在搜索栏中输入一个字母, DOM 树会默认跳到第一个符合条件的元素。 
```

> 对于经常使用搜索长语句的用户, 这个设定并不是很不理想。因为 DOM 树 可能会在您输入时，不断地跳跃搜寻 (e.g. `//div[@id="example"]`)，造成不必要的动效。 

Reviewer: ```suggestion
对于经常使用搜索长语句的用户，这个设定并不是很不理想。因为 DOM 树 可能会在您输入时，不断地跳跃搜寻 (e.g. `//div[@id="example"]`)，造成不必要的动效。 
```

> 对于用户, 特别时经常使用长查询语句的测试者, 这中行为是很不理想的. 因为您输入的长查询语句， DOM 树可能会发生数次跳跃 (e.g. `//div[@id="example"]`)。这个行为发生了不必需要的举动。 

Reviewer: ```suggestion
对于经常使用搜索长语句的用户, 这个设定并不是很不理想。因为 DOM 树 可能会在您输入时，不断地跳跃搜寻 (e.g. `//div[@id="example"]`)，造成不必要的动效。 
```

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/KgTTYf8XaKkHQ2udJc33.png", alt="DOM search.", width="800", height="505" %}

> 去到**Settings（设置）** > **Preferences（喜好）**, 禁止**Search as you type（输入即搜索）**. 通过这个改良, 搜索将会在点击 <kbd>Enter</kbd>才会发生。 

Reviewer: ```suggestion
打开 **设置** > **偏好设置**，禁止**即输即搜**。更改设置后，搜索将会在您点击<kbd>回车键</kbd>后才开始。 
```

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/HBLiQ5e60g5urU8UT5J7.png", alt="Search as you type setting.", width="800", height="449" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/b4643a4703b4a26945d1446eedc907ac81373e23 #}

Chromium 议题: [1344526](https://crbug.com/1344526)

> ## 在`align-content`的CSS flexbox 属性中显示`开始` 和 `结束`图标。 {: #flexbox } 

Reviewer: ```suggestion
## 在 `align-content` 的 CSS flexbox 属性中显示`start` 和 `end`图标。 {: #flexbox } 
```

`start` and `end` is code.

> 在 **样式**边栏中 , 点击 `display: flex` 或 `display: inline-flex` 旁边的图标打开 flexbox 编辑器。您可以利用图标来编辑 `align-content` 属性的 `start` 和 `end`。 

Reviewer: ```suggestion
在 **样式**边栏中，点击 `display: flex` 或 `display: inline-flex` 旁边的图标打开 flexbox 编辑器。您可以利用图标来编辑 `align-content` 属性的 `start` 和 `end`。 
```

> 在**Styles（风格）**界面中 , `使用在一个CSS 类中的display: flex` 或 `display: inline-flex`的`开始` and `结束`图标来编辑`align-content` 的属性 。 

Reviewer: ```suggestion
在 **样式**边栏中 , `使用在一个 CSS 类中的 `display: flex` 或 `display: inline-flex`的`start` and `end`图标来编辑 `align-content` 的属性 。 
```

pane is 边栏。Styles is 样式。Both of these are the term used in DevTools.

> {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fo10I2mt6bQ357itnYhl.png", alt="在`align-content`的CSS flexbox 属性中显示`开始` 和 `结束`图标。", width="800", height="424" %}

Reviewer: ```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fo10I2mt6bQ357itnYhl.png", alt="align-content 的 CSS flexbox 属性", width="800", height="424" %}
```

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fo10I2mt6bQ357itnYhl.png", alt="align-content 的 CSS flexbox 属性", width="800", height="424" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/ce2b426818106768d4e6d907cc1f4cd3b9636ca6 #}

Chromium 议题: [1139945](https://crbug.com/1139945)

 ## 其他的更新 {: #misc } 

>  在 **控制台 (Console)** 中显示正确的网页错误计数。在之前版本中, 网页错误计数并没有在清除控制台的讯息后被自动更新。([1343311](https://crbug.com/1343311)) 

Reviewer: ```suggestion
- 在 **控制台 (Console)** 中显示正确的网页错误计数。在之前版本中, 网页错误计数并没有在清除控制台的讯息后被自动更新。([1343311](https://crbug.com/1343311)) 
```

<!-- Content starts here -->

## 客制化 DevTools (开发者工具) 的键盘快捷键 {: #shortcuts }

您现在可以在 DevTools (开发者工具) 里客制化您喜爱的键盘快捷键。

> 在 ** 设置 （Settings）** > ** 快捷键 （Shortcuts）**, 将鼠标悬停在命令上并点击 ** 编辑 ** 的钢笔图示来客制化键盘快捷键。您也可以创建组合键（a.k.a. 多键点击快捷键）。

Reviewer: ```suggestion
在 ** 设置 （Settings）** > **快捷键 （Shortcuts）**, 将鼠标悬停在命令上并点击 ** 编辑 ** 的钢笔图示来客制化键盘快捷键。您也可以创建组合键（a.k.a. 多键点击快捷键）。
```

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/4853b34457f43e41ae9cebc7dfc97c0b734f463a #}
{# https://chromium.googlesource.com/devtools/devtools-frontend/+/029ac9db0b7e7d08945bcf7a16b407bde50183a1 #}

Chromium 议题: [1280398](https://crbug.com/1280398), [1226363](https://crbug.com/1226363)

## 高亮内存检查器里的 C/C++ 对象 {: #memory } 

<!-- The [Memory Inspector](/docs/devtools/memory-inspector/) highlights all the bytes of a C/C++ memory object. -->

辨识围绕在一个对象字节中的网络组件记忆体是一个用户痛点，您必须知道对象的体积以及从对象启动的时候开始计算字节数。 

有了这个新功能,  将帮助用户讲您指示网络组件记忆对象从围绕的记忆体中分离, 参看 [延伸 - 为了C/C++ 纠错而发生的机器检查器](/blog/memory-inspector-extended-cpp/) 来学习更多的改善。 

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/zqOv2zJTc8ucoeDmQiTo.png", alt="高亮记忆检查器里的C/C++对象。", width="800", height="527" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/d5f3befb47eaaa373d697b42dec6f179baf9d42c #}
{# https://chromium.googlesource.com/devtools/devtools-frontend/+/c4e6bdb4321cbc0b783647e855a616096beaabfd #}

Chromium 议题: [1336568](https://crbug.com/1336568)

## 支持 HAR 导入的完整启动器 （Initiator）的讯息。{: #har } 

> [HAR import] 的完整 ** 启动器 （Initiator） ** 讯息现在可以参见(/docs/devtools/network/reference/#save-as-har). 在之前的版本中, ** 网络 （Network）** 界面只显示启动器（Initiator) 的部分导入讯息。 

Reviewer: ```suggestion
[HAR import] 的完整 **启动器 （Initiator）** 讯息现在可以参见(/docs/devtools/network/reference/#save-as-har). 在之前的版本中, **网络 （Network）** 界面只显示启动器（Initiator) 的部分导入讯息。 
```

完整的启动器讯息将帮助开发者检索网络请求的源头并辨别和网络相关的问题。  

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/cthh3ZrpDwo4LJiaY4Uo.png", alt="导入 HAR 导入的完整启动器的资讯", width="800", height="376" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/3a659b0711f52a2e200395b85f16ed9f266d1571 #}

Chromium 议题: [1343185](https://crbug.com/1343185)

## 点击 ` 回车键 （Enter) `后开始 DOM 搜索。 {: #search-type } 

> 您现在可以禁止 ** 即输即搜 （Search as you type） ** 设置来启动点击 <kbd>Enter</kbd> 后开始DOM搜索。  

Reviewer: ```suggestion
您现在可以禁止 **即输即搜 （Search as you type）** 设置来启动点击 <kbd>Enter</kbd> 后开始DOM搜索。  
```

> 在 ** 元素 （Elements）**面板, 使用 <kbd>Control</kbd> 或 <kbd>Command</kbd> + <kbd>F</kbd> 来打开搜索栏，每当您在搜索栏中输入一个字母, DOM 树会默认跳到第一个符合条件的元素。 

Reviewer: ```suggestion
在**元素 （Elements）**面板, 使用 <kbd>Control</kbd> 或 <kbd>Command</kbd> + <kbd>F</kbd> 来打开搜索栏，每当您在搜索栏中输入一个字母, DOM 树会默认跳到第一个符合条件的元素。 
```

对于经常使用搜索长语句的用户, 这个设定并不是很不理想。因为 DOM 树 可能会在用户输入时, 不断地跳跃搜寻 (e.g. `//div[@id="example"]`)，造成不必要的动效。 

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/KgTTYf8XaKkHQ2udJc33.png", alt="DOM search.", width="800", height="505" %}

> 打开 ** 设置 (Settings) ** > ** 偏好设置 (Preferences) ** , 禁止 ** 即输即搜 (Search as you type) ** , 更改设置后，搜索将会在您点击<kbd>回车键</kbd>后才开始。 

Reviewer: ```suggestion
打开 **设置 (Settings)** > **偏好设置 (Preferences)** , 禁止 **即输即搜 (Search as you type)** , 更改设置后，搜索将会在您点击<kbd>回车键</kbd>后才开始。 
```

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/HBLiQ5e60g5urU8UT5J7.png", alt="Search as you type setting.", width="800", height="449" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/b4643a4703b4a26945d1446eedc907ac81373e23 #}

Chromium 议题: [1344526](https://crbug.com/1344526)

## 在 `align-content` 的 CSS flexbox 属性中显示`start` 和 `end`图标。 {: #flexbox } 

> 在 ** 样式 (Styles) **边栏中, 点击 `display: flex` 或 `display: inline-flex` 旁边的图标打开 flexbox 编辑器。

Reviewer: ```suggestion
在 **样式 (Styles)**边栏中, 点击 `display: flex` 或 `display: inline-flex` 旁边的图标打开 flexbox 编辑器。
```

您可以利用图标来编辑 `align-content` 属性的 `start` 和 `end`。 

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fo10I2mt6bQ357itnYhl.png", alt="align-content 的 CSS flexbox 属性", width="800", height="424" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/ce2b426818106768d4e6d907cc1f4cd3b9636ca6 #}

Chromium 议题: [1139945](https://crbug.com/1139945)

 ## 其他的更新 {: #misc } 

>  在 **控制台 (Console)** 中显示正确的网页错误计数。

Reviewer: ```suggestion
 - 在 **控制台 (Console)** 中显示正确的网页错误计数。([1343311](https://crbug.com/1343311)) 
```

>  在 **控制台 (Console)** 中显示正确的网页错误计数。

Reviewer: ```suggestion
 在 **控制台 (Console)** 中显示正确的网页错误计数。在之前版本中, 网页错误计数并没有在清除控制台的讯息后被自动更新。([1343311](https://crbug.com/1343311)) 
```

>  在 ** 控制台 (Console) ** 中显示正确的网页错误计数。

Reviewer: ```suggestion
 在 **控制台 (Console)** 中显示正确的网页错误计数。
```

>  在之前版本中, 网页错误计数并没有在清除控制台的讯息后被自动更新。

Reviewer: ```suggestion
```

> ([1343311](https://crbug.com/1343311)) 

Reviewer: ```suggestion
```

Chromium issue: [1344526](https://crbug.com/1344526)

## 在`align-content`的CSS flexbox 属性中显示`开始` 和 `结束`图标。 {: #flexbox } 

在**Styles（风格）**界面中 , `使用在一个CSS 类中的display: flex` 或 `display: inline-flex`的`开始` and `结束`图标来编辑`align-content` 的属性 。 

{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fo10I2mt6bQ357itnYhl.png", alt="在`align-content`的CSS flexbox 属性中显示`开始` 和 `结束`图标。", width="800", height="424" %}

{# https://chromium.googlesource.com/devtools/devtools-frontend/+/ce2b426818106768d4e6d907cc1f4cd3b9636ca6 #}

> Chromium issue: [1139945](https://crbug.com/1139945)

Reviewer 2: ```suggestion
议题 issue: [1139945](https://crbug.com/1139945)
```
